src/control/lib/spdk/ctests/nvme_control_ut.c

```diff
@@ -91,30 +139,30 @@ mock_spdk_nvme_detach_ok(struct spdk_nvme_ctrlr *ctrlr)
 static void
 test_nvme_discover_null_g_controllers(void **state)
 {
-	(void) state; /*unused*/
 	struct ret_t *ret;
 
-	assert_true(g_controllers == NULL);
+	(void) state; /*unused*/
```

Comment:
I've never run into errors with unused input params in tests before. Using a compiler flag we're not using in other unit tests?

Reply:
yes it's not absolutely necessary but given it's a fairly standard compiler warning don't think it's an issue to leave these in